ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -10,8 +10,15 @@
 if TYPE_CHECKING:  # pragma: no cover
```

Comment:
The changes made to remote config here don't seem necessary to the IP blocking. Why are they made in this PR?

Reply:
They are needed since it's the first RCM product registered outside of 1-click, thus need to register the feature.